docs/modules/usage/how-to/debugging.md

```diff
@@ -38,10 +25,31 @@ runs inside docker):
             "args": [
                 "openhands.server.listen:app",
                 "--reload",
+                "--reload-exclude",
+                "${workspaceFolder}/workspace",
                 "--port",
                 "3000"
             ],
             "justMyCode": false
+        },
+        {
+            "name": "D: CodeAct",
```

Comment:
This is just my "code": D for debugger, plus agent name. I have one for each agent. 😅 

We can probably name it something without abbreviations, e.g. "Debug CodeAct"

Reply:
I've put some more descriptions around this and isolated it from the rest of the json - given that the rest of the JSON can be run "as is"